src/main/resources/edu/cornell/kfs/paymentworks/businessobject/datadictionary/PaymentWorksFieldMapping.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans         
+	   http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+	<bean id="PaymentWorksFieldMapping" parent="PaymentWorksFieldMapping-parentBean" />
+	<bean id="PaymentWorksFieldMapping-parentBean" abstract="true" parent="BusinessObjectEntry">
+		<property name="businessObjectClass" value="edu.cornell.kfs.paymentworks.businessobject.PaymentWorksFieldMapping" />
+		<property name="objectLabel" value="Payment Works Field Mapping" />
+		<property name="titleAttribute" value="kfsFieldName" />
+		<property name="lookupDefinition" ref="PaymentWorksFieldMapping-lookupDefinition" />
+		<property name="attributes">
+			<list>
+				<ref bean="PaymentWorksFieldMapping-paymentWorksFieldMappingId" />
+				<ref bean="PaymentWorksFieldMapping-paymentWorksFieldName" />
+				<ref bean="PaymentWorksFieldMapping-kfsFieldName" />
+			</list>
+		</property>
+	</bean>
+	
+	<bean id="PaymentWorksFieldMapping-paymentWorksFieldMappingId" parent="PaymentWorksFieldMapping-paymentWorksFieldMappingId-parentBean" />
+	<bean id="PaymentWorksFieldMapping-paymentWorksFieldMappingId-parentBean" abstract="true" parent="AttributeDefinition">
+		<property name="name" value="paymentWorksFieldMappingId" />
+		<property name="label" value="Id" />
+		<property name="shortLabel" value="Id" />
+		<property name="forceUppercase" value="false" />
+		<property name="maxLength" value="10" />
+		<property name="control">
+			<bean parent="TextControlDefinition" p:size="20" />
```

Comment:
if the maxLength is 10, maybe the text control size should be 10 (or 12 if you want to add padding)?